aten/src/ATen/native/native_functions.yaml

```diff
@@ -3788,6 +3788,8 @@
   dispatch:
     CPU: legacy::cpu::_th_index_select
     CUDA: legacy::cuda::_th_index_select
+    SparseCPU: index_select_sparse
+    SparseCUDA: index_select_sparse
```

Comment:
I guess, the only possible explanation is that this has caused some of the generated files to change. Are you sure that you don't get this error locally? A matter of last resort would be to just manually apply the diff yourself, I guess.

@gchanan hurry please, I don't want to have checked in generated files anymore XD

Reply:
That is the thing, locally and in most CI cases there is no error.

Reply:
I figured it out: some of our jobs default to merge with master and that's what is causing the problem.

Reply:
Great, thanks!